app/docs/md/learn/practices/env-vars.md

```diff
@@ -35,7 +35,7 @@ API Routes are a Node.js process; `process.env` lists all environment variables.
 export async function get () {
   const env = process.env
   return { 
-    json: { env } 
+    json: { JSON.stringify(env) } 
```

Comment:
This would cause a parsing error as it would cause the following:

```javascript
json: { 
  "this is a string"
}
```

Which would cause a parsing error.

Reply:
Good catch @dranovsk. I've committed a fix https://github.com/enhance-dev/enhance.dev/commit/7fe57325cc638188d6c1b94d9edccfe8e52e5072 to take care of that issue and I'm going to close this PR. Thanks for your help.

Reply:
:+1: